### ISSUE TITLE
LSTM weights not optimised

### ISSUE THREAD

**Issue by timmeinhardt** [ikostrikov/pytorch-meta-optimizer#11]
The `LayerNormLSTMCell` modules initialised in the `MetaOptimizer` class are not properly registered as parameters of the `MetaOptimizer` model. Appending them to the `self.lstms` list:
https://github.com/ikostrikov/pytorch-meta-optimizer/blob/0154d4d4fc856163885a62bac06174311aa58caf/meta_optimizer.py#L27

will not add their trainable parameters to the model parameter list in:

https://github.com/ikostrikov/pytorch-meta-optimizer/blob/0154d4d4fc856163885a62bac06174311aa58caf/main.py#L63

If I am not mistaken, the current version will not train the LSTM weights at all. In general, I would suggest to restructure the initialisation and `MetaOptimizer.forward` method, but as a quick fix one could replace the entire `self.lstms` initialization block with this:

```
self.lstms = nn.Sequential(*[LayerNormLSTMCell(hidden_size, hidden_size)
                                                for _ in range(num_layers)])
```

**Comment by YanwenZhu** [comment]
This quick fix worked, thanks! By the way, have you been able to re-implement the experiments which were in the article using `MetaOptimizer`? For me the final loss of each epoch is really big, which is around 4 for best, and I cannot figure out what to do. Could you please give a little help?

**Comment by timmeinhardt** [comment]
@YanwenZhu  Sorry, but I did not reimplement the original experiments. I applied _Learning to learn_ to an unrelated problem.

**Comment by YanwenZhu** [comment]
@timmeinhardt Thanks anyway!
